documentation/docs/guides/unavailable-time.mdx

```diff
@@ -1,6 +1,3 @@
----
-sidebar_position: 8
----
```

Comment:
Are we certain we want to remove this? I see it in every guide 🤔

Reply:
Oh! I don't remember doing this on purpose 🤔 , nice catch!  ❤️